tests/server/testServerWithStorage.js

```diff
@@ -687,13 +687,13 @@ sjrk.storyTelling.server.testServerWithStorageDefs.rotateImageFromExifTests = fu
         { fileName: "", options: null, expectedResolution: false },
         { fileName: "", options: "", expectedResolution: false },
         { fileName: "correctOrientation.jpg", options: null, expectedResolution: true },
-        { fileName: "incorrectOrientation.jpeg", options: null, expectedResolution: true },
+        { fileName: "incorrectOrientation.jpeg", options: null, expectedResolution: true, expectedOrientation: 1 },
```

Comment:
Thanks for the improvement, @BlueSlug. It would be more helpful to further improve the test to show what processing results you expect from sjrk.storyTelling.server.rotateImageFromExif() when it deals with different media types. You have quite comprehensive test files with images, audio, video, but the test for them is mostly to show they are all resolved by this function. The test doesn't show what you expect this function does to them? For example, if you expect this function does nothing to audios, videos and images with correct orientations, add steps to show orientation values, or some other indicators, of these files stay unchanged before and after the rotation, while orientation values do change for images with incorrect orientations.